spec/test-outputs/www-integration.out.vcl

```diff
@@ -426,6 +454,32 @@ sub vcl_deliver {
   }
   # End dynamic section
 
+if (req.http.Cookie ~ "cookies_policy") {
+  if (req.http.Cookie:cookies_policy ~ "%22usage%22:true" && req.http.User-Agent !~ "^GOV\.UK Crawler Worker") {
+    declare local var.variant STRING;
+    set var.variant = req.http.GOVUK-ABTest-StartABusinessSegment;
+
+    declare local var.sab_expiry TIME;
+    if (var.variant == "C") {
+      # If they have opted out of the banner or have visited the SaB checker
+      # Expire in 26 weeks (approx 6 months)
+      set var.sab_expiry = time.add(now, std.integer2time(15724800));
+    } else {
+      # Otherwise, expire in half an hour, approx this session
+      set var.sab_expiry = time.add(now, std.integer2time(1800));
+    }
+
+    if (req.http.GOVUK-ABTest-IsStartABusinessPage == "true" && var.variant == "A") {
```

Comment:
You probably want this to be an else if

Reply:
Do you mean something like this? 
```
    if (var.variant == "C") {
      # If they have opted out of the banner or have visited the SaB checker
      # Expire in 26 weeks (approx 6 months)
      set var.sab_expiry = time.add(now, 26w);
    } else if (req.http.GOVUK-ABTest-IsStartABusinessPage == "true" && var.variant == "A") {
      # Otherwise, expire in half an hour, approx this session
      set var.sab_expiry = time.add(now, 30m);
      # If the user is in variant A and has seen a SaB page then we need to move them to B
      # so they see the banner on the next SaB page they hit.
      set var.variant = "B";
    } else {
      # Otherwise, expire in half an hour, approx this session
      set var.sab_expiry = time.add(now, 30m);
    }
```

Reply:
I see you've already updated this 👍